app/presenters/licence-versions/view.presenter.js

```diff
@@ -22,12 +22,35 @@ function go(licenceVersion) {
       href: `/system/licences/${licence.id}/history`,
       text: 'Go back to history'
     },
+    changeType: licenceVersion.administrative ? 'no licence issued' : 'licence issued',
+    notes: _notes(licenceVersion),
     pageTitle: `Licence version starting ${formatLongDate(licenceVersion.startDate)}`,
     pageTitleCaption: `Licence ${licence.licenceRef}`,
-    changeType: licenceVersion.administrative ? 'no licence issued' : 'licence issued'
+    reason: _reason(licenceVersion)
   }
 }
 
+function _notes(licenceVersion) {
+  const notes = licenceVersion.$notes()
+
+  return notes.length > 0 ? notes : null
+}
+
+function _reason(licenceVersion) {
+  const reason = licenceVersion.$reason()
+  const createdBy = licenceVersion.$createdBy()
+
+  if (!reason) {
+    return null
+  }
+
+  if (!createdBy) {
+    return `${reason} created on ${formatLongDate(licenceVersion.createdAt)}`
+  }
```

Comment:
We'll always have a `createdOn` value.

So, if `reason` nor `createdBy` exist, we could still display "Created on 2 December 2025" as a minimum.